Source/Core/Core/Boot/Boot.cpp

```diff
@@ -388,6 +388,11 @@ bool CBoot::BootUp()
       PowerPC::DBATUpdated();
       PowerPC::IBATUpdated();
 
+      // Because there is no TMD to get the requested system (IOS) version from,
+      // we default to IOS58, which is the version used by the Homebrew Channel.
+      if (dolLoader.IsWii())
+        SetupWiiMemory(0x000000010000003a);
```

Comment:
>[18:49:45] \<booto\> JosJuice: I think it was recent & shared common values for the various 31xx values, but if there's something else more appropriate for w/e reason, changing it is no big deal (maybe this should be configurable via the wii panel?)

Since booto does not mind changing it to something more appropriate), I have changed it to 58.